functional-tests/src/test/java/org/infinispan/online/service/caching/CachingServiceTest.java

```diff
@@ -29,30 +32,60 @@
    @ArquillianResource
    URL restService;
 
+   @Named("caching-service-app-memcached")
+   @ArquillianResource
+   URL memcachedService;
+
    @ArquillianResource
    OpenShiftClient openShiftClient;
 
    ReadinessCheck readinessCheck = new ReadinessCheck();
    HotRodTester hotRodTester = new HotRodTester();
    RESTTester restTester = new RESTTester();
+   MemcachedTester memcachedTester = new MemcachedTester();
 
    @Before
    public void before() {
       readinessCheck.waitUntilAllPodsAreReady(openShiftClient, 30, TimeUnit.SECONDS);
    }
 
    @Test
-   public void should_default_cache_be_accessible_via_hot_rod() throws IOException {
-      hotRodTester.testBasicEndpointCapabilities(hotRodService);
+   @Ignore // TODO: remove when https://issues.jboss.org/browse/ISPN-8536 is resolved
```

Comment:
Since @ryanemerson merged https://github.com/jboss-container-images/jboss-dataservices-image/pull/33, this might be uncommented.

Reply:
@slaskawi  This is still failing on my laptop sometimes. I've added more logs to the https://issues.jboss.org/browse/ISPN-8536 .
If it's passing consistently in your environment, I suppose I could enable the test and disable it when I run the tests manually, until we figure out why is it failing on my machine.

Reply:
I rebased the latest changes v dataservices-dev branch and erased all local docker images before the test: docker images -q |xargs docker rmi --force

Reply:
Yes, please do so... The issue needs to be visible or otherwise it probably won't be fixed.